### PR TITLE
Fix transition manager tests

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

File: test/src/utils/transition-manager.spec.js [hunk 1]
```diff
@@ -404,6 +406,9 @@ test('TransitionManager#TRANSITION_EVENTS', t => {
         testCase.shouldChange[mode].transitionEasing,
         'transitionEasing match'
       );
+
+      // Clean up
+      transitionManager._endTransition();
```
Comment (xintongxia) [inline test/src/utils/transition-manager.spec.js]: Why this is needed to be called in tests? Is this a step that whoever uses TransitionManager's also need to do?

File: test/src/utils/transition-manager.spec.js [hunk 2]
```diff
@@ -395,7 +397,7 @@ test('TransitionManager#TRANSITION_EVENTS', t => {
       // testing easing function
       let testEasingFunc = testCase.input[ti].transitionEasing;
       if (mode === TRANSITION_EVENTS.UPDATE) {
-        const completion = (time[1] - time[0]) / testCase.input[ti].transitionDuration;
+        const completion = 100 / testCase.input[ti].transitionDuration;
```
Comment (xintongxia) [inline test/src/utils/transition-manager.spec.js]: Where did you get this number `100`?

Comment (Pessimistress) [reply test/src/utils/transition-manager.spec.js]: https://github.com/uber/react-map-gl/pull/798/files#diff-a02c4dd2c586d5f96eeeb6581028fb59R376

File: test/src/utils/transition-manager.spec.js (hunk 1 above)
Comment (Pessimistress) [reply test/src/utils/transition-manager.spec.js]: We provided an external timer so the animation never ends. The test cannot end if there is a pending animation frame requested.

**Pull request merged**